src/mame/layout/dfs500.lay

```diff
@@ -0,0 +1,286 @@
+<?xml version="1.0"?>
+<!--
+license:CC0
+-->
+<mamelayout version="2">
+	<element name="digit" defstate="255">
+		<led7seg>
+			<color red="1" green="0.0" blue="0.0" />
+		</led7seg>
+	</element>
+
+	<element name="led" defstate="1">
+		<disk state="1">
+			<color red="1" green="0.4" blue="0.4" />
+		</disk>
+		<disk state="0">
+			<color red="0.5" green="0.5" blue="0.5" />
+		</disk>
+	</element>
+
+	<element name="squared_led" defstate="1">
+		<rect state="1">
+			<color red="0.4" green="1.0" blue="0.4" />
+		</rect>
+		<rect state="0">
+			<color red="0.05" green="0.3" blue="0.05" />
+		</rect>
+	</element>
+
+	<element name="rect" defstate="0">
+		<rect state="0">
+			<color red="0.1" green="0.05" blue="0.05" />
+		</rect>
+	</element>
+
+	<element name="controlpanel"><image file="cpanel.png" /></element>
+
+	<!-- Snap Shot -->
+
+	<!-- Lighting -->
+	<element name="lighting_spot"  defstate="1"><image file="lighting_spot.png" state="1"/></element>
+	<element name="lighting_line"  defstate="1"><image file="lighting_line.png" state="1"/></element>
+	<element name="lighting_plane" defstate="1"><image file="lighting_plane.png" state="1"/></element>
+	<element name="lighting_width_wide"     defstate="1"><image file="lighting_width_wide.png" state="1"/></element>
+	<element name="lighting_width_medium"   defstate="1"><image file="lighting_width_medium.png" state="1"/></element>
+	<element name="lighting_width_narrow"   defstate="1"><image file="lighting_width_narrow.png" state="1"/></element>
+	<element name="lighting_intensity_high"    defstate="1"><image file="lighting_intensity_high.png" state="1"/></element>
+	<element name="lighting_intensity_medium"  defstate="1"><image file="lighting_intensity_medium.png" state="1"/></element>
+	<element name="lighting_intensity_low"     defstate="1"><image file="lighting_intensity_low.png" state="1"/></element>
```

Comment:
You can’t require external files with internal artwork.  You’ll need to draw some basic buttons and indicators using the built-in components (mostly rect, disk and text).  There are plenty of examples you can learn from.

Reply:
done